Makefile

```diff
@@ -109,11 +109,13 @@ clean:
 	rm -f man/avocado.1
 	rm -f man/avocado-rest-client.1
 	rm -rf docs/build
+	rm -rf avocado.egg-info
```

Comment:
Actually shouldn't we remove the `egg-info` on `make clean`? I think that would make sense.